RELEASES.md

```diff
@@ -1,3 +1,122 @@
+Version 1.37.0 (2019-08-15)
+==========================
+
+Language
+--------
+- [`#[must_use]` will now warn if the type is in a tuple and unused.][61100]
+- [`#[must_use]` will now warn if the type is in a `Box` and unused.][62228]
+- [`#[must_use]` will now warn if the type is in a array and unused.][62235]
+- [You can now use the `cfg` and `cfg_attr` attributes on
+  generic parameters.][61547]
+- [You can now use enum variants through type alias.][61682] e.g. You can
```

Comment:
This arguably is the most important language change so I would put it first. Then I would mention consts, then Repr align. Also, I would use an example with Self here since that is likely the most common use.

Reply:
I think that arbitrary reordering does not provide much benefit in the RELEASES.md document. This is really the point of the blog post to highlight specific features and guide a user through a new release.

Reply:
The reordering is not arbitrary, it's based on the relative importance of the features by estimating how many users will be affected and benefit from each one.

> This is really the point of the blog post to highlight specific features and guide a user through a new release.

If so then we can dispense with examples in the release notes altogether. I think the blog post does a good job of talking in-depth about the most important bits but some users will read the release notes as well and we should use that opportunity wisely.

Reply:
> The reordering is not arbitrary, it's based on the relative importance of the features by estimating how many users will be affected and benefit from each one.

This would be important if the entire section was longer, but it isn't. It fits entirely on the page on all devices except mobile. Personally I find this to be pointless busy work, and I'm not going to do it. If you think it's that important feel free to make a separate PR.